R/CHANGED-pre-msa-tree.R

```diff
@@ -203,6 +208,14 @@ addLeaves2Alignment <- function(aln_file = "",
 #' @export
 #'
 #' @examples
+#' # Example usage of the addName function
+#' data <- data.frame(
+#'   AccNum = c("ACC123", "ACC456"),
+#'   Species = c("Homo sapiens", "Mus musculus"),
+#'   Lineage = c("Eukaryota>Chordata", "Eukaryota>Chordata")
+#' )
+#' enriched_data <- addName(data)
+#' print(enriched_data)
```

Comment:
In general, it is unnecessary to use `print()` in this manner. For brevity in our example code and to avoid potential side effects, we should use:
```
data <- data.frame(
  AccNum = c("ACC123", "ACC456"),
  Species = c("Homo sapiens", "Mus musculus"),
  Lineage = c("Eukaryota>Chordata", "Eukaryota>Chordata")
)
enriched_data <- addName(data)
enriched_data
```

Reply:
Thanks @the-mayer